tests/sfp_test.py

```diff
@@ -122,8 +122,88 @@ def test_sfp_eeprom(self):
         expected = "Ethernet200: SFP EEPROM Not detected"
         assert result_lines == expected
 
+    def test_sfp_presence_with_ns(self):
```

Comment:
The command has asic name when testing single asic . Is this a negative test case ?

Reply:
removed, this was my initial idea to club both single/multi asic in a single testcase. Later made a new multi-ASIC testcase but missed to remove this. thanks for pointing out.